src/ast/ast_build_filter_tree.c

```diff
@@ -192,8 +192,9 @@ static FT_FilterNode *_convertInlinedProperties(const AST *ast, const cypher_ast
 	for(uint i = 0; i < nelems; i ++) {
 		// key is of type CYPHER_AST_PROP_NAME
 		const char *prop = cypher_ast_prop_name_get_value(cypher_ast_map_get_key(props, i));
-		AR_ExpNode *lhs = AR_EXP_NewVariableOperandNode(alias, prop);
-		lhs->operand.variadic.entity_alias_idx = IDENTIFIER_NOT_FOUND;
+
+		AR_ExpNode *entity = AR_EXP_NewVariableOperandNode(alias);
```

Comment:
`entity` is a redeclaration of a function argument.

Reply:
👍